lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -664,17 +664,23 @@ SwiftLanguageRuntimeImpl::emplaceClangTypeInfo(
     m_clang_type_info.insert({clang_type.GetOpaqueQualType(), llvm::None});
     return nullptr;
   }
+  auto byte_align = *bit_align / 8;
+  // The stride is the size rounded up to alignment.
+  size_t byte_stride = *byte_size + byte_align - 1;
+  byte_stride -= (byte_stride % byte_align);
```

Comment:
I think this could use [`llvm::alignTo`](https://llvm.org/doxygen/namespacellvm.html#ad39b4bf2c074c99f4e9b79c0698eeebe).